# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/libvolk/hotfix-camke.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- volk-3.3.0/CMakeLists.txt.vanilla	2026-07-30 16:44:33.444831566 +0300
+++ volk-3.3.0/CMakeLists.txt	2026-07-30 17:03:26.195910714 +0300
@@ -386,12 +388,12 @@
 ########################################################################
 # Add tests
 ########################################################################
-add_subdirectory(tests)
+#add_subdirectory(tests)
 
 ########################################################################
 # And the utility apps
 ########################################################################
-add_subdirectory(apps)
+#add_subdirectory(apps)
 option(ENABLE_MODTOOL "Enable volk_modtool python utility" True)
 if(ENABLE_MODTOOL)
     add_subdirectory(python/volk_modtool)
